tests/snmpsim/coriant.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.42229.6.22
```

Comment:
sysDescr is not relevant. Since Coriant TNMS is a proxy.
On my TNMS we have : 
 snmpwalk -v2c -c OsB24 par-eqx-tnms-01:50161 1.3.6.1.2.1.1.1.0
iso.3.6.1.2.1.1.1.0 = STRING: "Linux version 3.10.0-327.el7.x86_64"

But you can have also a Windows. So sysDescr is not usuable.